ctapipe/core/traits.py

```diff
@@ -636,7 +643,7 @@ class IntTelescopeParameter(TelescopeParameter):
 
     def __init__(self, **kwargs):
         """Create a new IntTelescopeParameter"""
-        super().__init__(trait=Int(), **kwargs)
+        super().__init__(trait=Integer(), **kwargs)
```

Comment:
Why do we have both `Int` and `Integer`?

Reply:
These are aliases in traitlets, exactly the same object....